src/ui/pages/Menu/components/ConnectWallet/ConnectWallet.tsx

```diff
@@ -182,6 +201,51 @@ const ConnectWallet = forwardRef<ConnectWalletOptionRef, object>(
       history.push(TabsRoutePath.IDENTIFIERS);
     };
 
+    const getDisplayConnection = useCallback(
```

Comment:
Do you mean instead of save user input `meerkatId` as a pending meerkatId, we should save it as a connecting connection and create pending flag ?

Reply:
No need for the pending flag.

I just meant if whenever we store the pending `meerkatId`, if we have to search over the list of connections for a match - it would've been easier to just store the entire pendingConnection in Redux instead and if it's set, display the entire connection without searching.